book/accessibility.md

```diff
@@ -2440,9 +2445,7 @@ class Browser:
 
 By the way, the acronym `a11y` in `a11y_node`, with an "a", the number 11, and
 a "y", is a common shorthand for the word "accessibility".[^why-11] The
-`hit_test` function I'm calling is the same one we wrote
-[earlier in Section 7.3](chrome.md#click-handling) to handle clicks, except
-of course that it's searching a different tree:
+`hit_test` function recurses over the accessibility tree:
```

Comment:
It is not similar! An earlier draft of the book used similar code, but now the A11y hit-test method is recursive while layout hit-test isn't.